paradise_papers_django/search/models/helpers.py

```diff
@@ -0,0 +1,73 @@
+from neomodel import db
+from . import (
+    Entity,
+    Address,
+    Intermediary,
+    Officer,
+    Other
+)
+
+MODELENTITIES = {
+'entity': Entity.Entity,
+'address': Address.Address,
+'intermediary': Intermediary.Intermediary,
+'officer': Officer.Officer,
+'other': Other.Other
+}
+
+# Queries Functions
+def get_all_countries():
+    query = "MATCH (n) WHERE NOT n.countries CONTAINS ';' RETURN DISTINCT 'node' as entity, n.countries AS countries UNION ALL MATCH ()-[r]-() WHERE EXISTS(r.countries) RETURN DISTINCT 'relationship' AS entity, r.countries AS countries"
+    results = db.cypher_query(query)
+    return results
+
+def filter_by_name_and_contry(entity, name, country):
+    country_selected = (country, '')[country == 'allcountry']
+    entity_node = MODELENTITIES[entity].nodes.filter(name__icontains=name).filter(countries__icontains=country_selected)
```

Comment:
Need to find a way to refactor this function to return a group of results instead of one